### ISSUE TITLE
[Effect Suggestion] Mater Driving

### ISSUE THREAD

**Issue by RosyMiranto18** [gta-chaos-mod/ChaosModV#1747]
**Name of effect**
Mater Driving

**Duration if timed effect (long or short timed?)**
Short

**Effect Description**
Basically, it's a "Help, my S Key is Stuck" Effect or the player equivalent of Everyone Drives Backwards. To get around it while walking, you can press Look Back to see your face and keep walking forward. If you're driving, well... Prepare to channel your inner Mater and start driving in reverse.

**Comment by DEATHCON1** [comment]
But what if you hold the button to go forward too? You'll just be doing a burnout...

**Comment by RosyMiranto18** [comment]
Yes, which is why you shouldn't do that, unless if Help My W Key is stuck is also on.
